### PR TITLE
Add the ability to change elasticsearch admin username in logs-concentrator

### PR DESCRIPTION
**Pull request by KorvinSzanto**
This PR makes it possible to configure an elasticsearch username as part of the lagoon-logs-concentrator chart values.yml file.

### CODE REVIEW

**Comment by KorvinSzanto** [comment]
There is still work to do in the lagoon-core chart as well as in https://github.com/uselagoon/lagoon to remove the 'admin' default

File: charts/lagoon-logs-concentrator/templates/secret.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ metadata:
     {{- include "lagoon-logs-concentrator.labels" . | nindent 4 }}
 stringData:
   FORWARD_SHARED_KEY: {{ required "A valid .Values.forwardSharedKey required!" .Values.forwardSharedKey }}
+  LOGSDB_ADMIN_USER: {{ default "admin" .Values.elasticsearchAdminUser }}
```
Comment (smlx) [inline charts/lagoon-logs-concentrator/templates/secret.yaml:10]: Minor style thing, but could we change this to have the default value in the `values.yaml`, and just reference `{{ .Values.elasticsearchAdminUser }}` here? This is the style we have used elsewhere in the chart.

**Review by smlx — CHANGES_REQUESTED** [review]
Needs a very minor change but otherwise looks good!

Also needs a rebase and to bump the minor version number of the `lagoon-logs-concentrator` chart.

**Comment by smlx** [comment]
@KorvinSzanto just checking if you're still working on this PR?

**Comment by KorvinSzanto** [comment]
I can take a look tomorrow, thanks for the heads up

**Comment by KorvinSzanto** [comment]
This should be good  to go, let me know if I missed anything or did anything incorrectly.

**Comment by smlx** [comment]
Thanks!

**Pull request merged**